src/main/java/Task.java

```diff
@@ -0,0 +1,28 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+        System.out.println("Got it. I've added this task: ");
+        //System.out.print(toString());
```

Comment:
Consider removing commented parts of the code if they are not in use

Reply:
This is a good suggestion, thank you for catching this!